pkg/artifactory/resource_artifactory_local_rpm_repository.go

```diff
@@ -40,6 +37,18 @@ func resourceArtifactoryLocalRpmRepository() *schema.Resource {
 				"the group definitions as part of the calculated RPM metadata, as well as automatically generating a " +
 				"gzipped version of the group files, if required.",
 		},
+		"primary_keypair_ref": {
```

Comment:
I feel like this may be duplicate code from somewhere else. @maheshjfrog do you know?

Reply:
primary_keypair_ref and secondary_keypair_ref fields are specified in other repo resources too. yes. Eg: RPM, debian. 
Alpine repo type has primary_keypair_ref but not secondary_keypair_ref.

Reply:
Ok, I thought for sure there was one other place that had both - I mean, in the ticket her references it